awxkit/awxkit/api/pages/execution_environments.py

```diff
@@ -33,12 +33,13 @@ def create_payload(self, name='', organization=Organization, **kwargs):
         payload.ds = DSAdapter(self.__class__.__name__, self._dependency_store)
         return payload
 
-    def payload(self, name='', image=None, organization=None, credential=None, **kwargs):
+    def payload(self, name='', image=None, organization=None, credential=None, pull=None, **kwargs):
         payload = PseudoNamespace(
             name=name or "EE - {}".format(random_title()),
             image=image or random_title(10),
             organization=organization.id if organization else None,
             credential=credential.id if credential else None,
+            pull=pull if pull else None,
```

Comment:
I think this can just be `pull=pull` since it _is_ `None` by default.

Reply:
credential is none by default as well, I just was matching up, but I can change it

Reply:
It's passing along the _id_ of the credential if there is one:

```
credential=credential.id if credential else None
```

Here you are just passing along `pull`, so this conditional is unnecessary.

Reply:
done